util/docker/README.md

```diff
@@ -0,0 +1,22 @@
+# Docker utility files
+
+## dev.sh
+
+Run the file-watching, auto-compiling dev server in a Docker instance under the
+name `wptd-dev-instance`.
+
+## exec.sh
+
+Execute a command in the context of `wptd-dev-instance`.
+
+## su_exec.sh
+
+Execute a command as root in the context of `wptd-dev-instance`.
+
+## jenkins_container.sh
+
+Launch container used by Jenkins CI to perform WPT test runs.
```

Comment:
This script is meant to run in Travis CI, not Jenkins CI. It's an e2e test of the test runner so we know that when something lands on master in w3c/wptdashboard, it will (hopefully) work when Jenkins pulls the image and runs it.

Reply:
Ah. I see.